features/config/rest/impl/src/main/java/org/opennms/features/config/rest/impl/ConfigManagerRestServiceImpl.java

```diff
@@ -94,58 +95,79 @@ public Response getOpenApiSchema(String configName, String acceptType){
     }
 
     @Override
-    public Response listConfigs() {
+    public Response getConfigIds(String configName) {
         try {
-            return Response.ok(configurationManagerService.getConfigNames()).build();
-        } catch (IOException | RuntimeException e) {
-            LOG.error(e.getMessage());
-            return Response.serverError().build();
+            Set<String> ids = configurationManagerService.getConfigIds(configName);
+            return Response.ok(ids).build();
+        } catch (IOException e) {
+            return this.generateSimpleMessageResponse(Response.Status.BAD_REQUEST, e.getMessage());
         }
     }
 
-    /**
-     * get or create a fake schema and return
-     *
-     * @param configName
-     * @return
-     */
     @Override
-    public ConfigSchema getSchema(String configName) {
+    public Response getConfig(String configName, String configId) {
```

Comment:
I wonder if we should have a 404 if a config can't be found?

Reply:
done